WORKSPACE

```diff
@@ -0,0 +1,2 @@
+workspace(name = "com_github_google_flatbuffers")
```

Comment:
is this file strictly needed? always looking for ways to not clutter the root dir :)

Reply:
Yup, Bazel requires a WORKSPACE file at the root directory:
https://docs.bazel.build/versions/master/tutorial/cpp.html#build-with-bazel